cmd/update-rules/main.go

```diff
@@ -126,7 +128,7 @@ func UpdateRules(rules *config.RepositoryRules, branch, goVer string) {
 		var newBranchRule config.BranchRule
 		// find the mainBranch rules
 		for _, br := range r.Branches {
-			if br.Name == "master" {
+			if br.Name == MainBranchName {
```

Comment:
As a follow up, should we prepare this to be both `master` and `main` to be ready when changing k/k branch name to `main`?

Reply:
sounds good as follow up PR.